recipes/promod3/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "ProMod3" %}
+{% set version = "3.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # url: https://git.scicore.unibas.ch/schwede/{{ name }}/-/archive/{{ version }}/{{ name|lower }}-{{ version }}.tar.gz
+  url: https://git.scicore.unibas.ch/schwede/ProMod3/-/archive/conda/ProMod3-conda.tar.gz
+  sha256: 91fd8b36302c026a0f4601a5e7ca8820bbc02fdbee272f0911a753ab83a7f8d6
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name|lower, max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - cmake
+    - make
+    - pkg-config
+  host:
+    - python {{ python }}
+    - libboost-devel >=1.86,<1.87
+    - libboost-python-devel >=1.86,<1.87
+    - openmm >=7
+    - eigen
+    - openstructure >=2.11.0
+  run:
+    - python {{ python }}
+    - {{ pin_compatible("openmm") }}
+
+test:
+  source_files:
+    - doc/tests/data/1crn.fasta
+    - doc/tests/data/1crn_cut.pdb
+    - doc/tests/data/1CRNA.hhm
+  commands:
+    - pm 2>&1 | grep -q Usage
+
```

You can check if the Python module was installed successfully by trying `import promod3`.

```suggestion
  imports:
    - promod3
```